.devcontainer/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM centos:7.9.2009
```

Comment:
Based on my experience developing a `dev container` for [OpenDAL](https://github.com/apache/incubator-opendal), I recommend using `mcr.microsoft.com/devcontainers/rust:bullseye` and customizing it with `post_create.sh`. For more information, please visit https://github.com/apache/incubator-opendal/tree/main/.devcontainer.


- All images are cached so users don't need to fetch them.
- Those images are well-maintained and updated.
- Those images handles many suck things related to users permission which is not easy (we tried and failed at OpenDAL).

Reply:
Thank you, let me try